app/src/main/res/values-v21/styles.xml

```diff
@@ -1,5 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+
+  <style name="OppiaThemeWithoutActionBar" parent="Theme.AppCompat.Light.NoActionBar">
+    <item name="colorPrimary">@color/colorPrimary</item>
+    <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+    <item name="colorAccent">@color/colorAccent</item>
+    <item name="windowActionBar">false</item>
+    <item name="windowNoTitle">true</item>
+    <item name="android:windowAnimationStyle">@style/Animation.Activity</item>
+  </style>
+  <!-- Base style for animations.  This style specifies no animations. -->
+  <style name="Animation" />
+  <!-- Standard animations for a full-screen window or activity. -->
+  <style name="Animation.Activity">
+    <item name="android:activityOpenEnterAnimation">@anim/slide_right_in</item>
```

Comment:
Can you rename this to slide_in_right for consistency.

Reply:
Done